src/Client.js

```diff
@@ -969,7 +1025,88 @@ class Client extends EventEmitter {
             return await window.WWebJS.getChat(chatId);
         }, chatId);
 
-        return ChatFactory.create(this, chat);
+        return chat
+            ? ChatFactory.create(this, chat)
+            : chat;
```

Comment:
why return the chat if it not exist?

Reply:
@shirser121 

> why return the chat if it not exist?

[Fixed](https://github.com/pedroslopez/whatsapp-web.js/pull/2620/commits/8ed8626b268e99b2b5ade2b46b654a33334811cf), now returns `undefined` in a case if a `chat` is falsy